client/Dockerfile

```diff
@@ -3,7 +3,6 @@ WORKDIR '/app'
 COPY package.json package-lock.json ./
 RUN npm install
 COPY . .
-RUN npm run build
```

Comment:
While `npm run build` is not required for the backend, I believe this one is required for the client; @anniegiang can you confirm?

Reply:
Yes we definitely need to build the frontend for it to production ready